client/src/annoMatrix/views.ts

```diff
@@ -95,35 +129,43 @@ class AnnoMatrixView extends AnnoMatrix {
     return newAnnoMatrix;
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-  addEmbedding(colSchema: any) {
+  addEmbedding(colSchema: EmbeddingSchema): AnnoMatrix {
     const newAnnoMatrix = this._clone();
+    // @ts-expect-error ts-migrate --- TODO revisit: see comments on L46
     newAnnoMatrix.viewOf = this.viewOf.addEmbedding(colSchema);
     newAnnoMatrix.schema = newAnnoMatrix.viewOf.schema;
     return newAnnoMatrix;
   }
 }
 
 class AnnoMatrixMapView extends AnnoMatrixView {
+  mapFn: MapFn;
+
   /*
-	A view which knows how to transform its data.
-	*/
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'viewOf' implicitly has an 'any' type.
-  constructor(viewOf, mapFn) {
+    A view which knows how to transform its data.
+    */
+  constructor(viewOf: AnnoMatrix, mapFn: MapFn) {
     super(viewOf);
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-    (this as any).mapFn = mapFn;
+    this.mapFn = mapFn;
   }
 
-  // @ts-expect-error ts-migrate(2416) FIXME: Property '_doLoad' in type 'AnnoMatrixMapView' is ... Remove this comment to see the full error message
-  async _doLoad(field, query) {
+  async _doLoad(
+    field: Field,
+    query: Query
+  ): Promise<[WhereCache | null, Dataframe]> {
+    // @ts-expect-error ts-migrate --- TODO revisit:
+    // `this.viewOf`: Object is possibly 'undefined'.
     const df = await this.viewOf._fetch(field, query);
-    // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'colData' implicitly has an 'any' type.
-    const dfMapped = df.mapColumns((colData, colIdx) => {
+    // @ts-expect-error ts-migrate --- TODO revisit:
+    // `df`: Object is possibly 'undefined'.
+    const dfMapped = df.mapColumns((colData: string[], colIdx: number) => {
```

Comment:
colData is not a string[], but any underlying dataframe array type.  In the new Dataframe PR, it is `DataframeValueArray`

Reply:
Updated [views.ts:_doLoad](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/views.ts#L146).